templates/navigation/_dropdown.html

```diff
@@ -0,0 +1,200 @@
+              {% endfor %}
+              {% if nav_section.secondary_links %}
+                {% for links_section in nav_section.secondary_links %}
+                  <li class="p-navigation--list-heading">
+                    <h2 class="p-muted-heading is-muted">{{ links_section.title }}</h2>
+                  </li>
+                  {% for link in links_section.links %}
+                    <li>
+                      <a class="p-navigation--list-link" href="{{ link.url }}">{{ link.title }}</a>
+                    </li>
+                  {% endfor %}
+                {% endfor %}
+              {% endif %}
+            </ul>
+          </li>
+        {% endfor %}
+      {% endif %}
+      <!-- Check for CTA link -->
+      {% if section.cta and section.cta_link %}
+        <li>
+          <a class="p-navigation__dropdown-item p-heading--2"
+             style="margin-bottom:0"
+             href="{{ section.cta_link }}">{{ section.cta }}</a>
+        </li>
+      {% endif %}
+      <!-- Loop through each of the main lists -->
+      {% if section.main_lists %}
+        {% for main_list in section.main_lists %}
+          <li class="p-navigation--list-heading">
+            <h2 class="p-muted-heading">{{ main_list.title }}</h2>
+          </li>
+          {% for link in main_list.link_list %}
+            <li>
+              <a class="p-navigation__dropdown-item" href="{{ link.url }}">{{ link.title }}</a>
+            </li>
+          {% endfor %}
+        {% endfor %}
+      {% elif section.primary_links %}
+        {% for link in section.primary_links.links %}
+          <li>
+            <a class="p-navigation__dropdown-item" href="{{ link.url }}">
+              {% if id == "careers" %}
+                <strong>{{ link.title }}</strong>
+                <br />
+                <span class="is-muted js-available-roles" data-id="{{ link.title | slug }}"><i class="p-icon--spinner u-animation--spin"></i></span>
+            {% else %}
+              {{ link.title }}
+            {% endif %}
+          </a>
+        </li>
+      {% endfor %}
+    {% endif %}
+    {% if section.secondary_links %}
+      <li class="p-navigation--list-heading">
+        <h2 class="p-muted-heading">{{ section.secondary_links.title }}</h2>
+      </li>
+      {% for link in section.secondary_links.links %}
+        <li>
+          <a class="p-navigation--list-link" href="{{ link.url }}">{{ link.title }}</a>
+        </li>
+      {% endfor %}
+    {% elif section.preview_secondary_links %}
+      <li class="p-navigation--list-heading">
+        <h2 class="p-muted-heading">(WAITING ON DESIGN) {{ section.preview_secondary_links.title }}</h2>
+      </li>
+      {% for link in section.preview_secondary_links.links %}
+        <li>
+          <a class="p-navigation--list-link" href="{{ link.url }}">(WAITING ON DESIGN) {{ link.description }}</a>
+        </li>
+      {% endfor %}
+    {% endif %}
+  </ul>
+</div>
+<div class="p-navigation__dropdown-content">
```

Comment:
Could you please clarify if the placeholder texts for this nav dropdown will be replaced for the actual desktop view?

Reply:
That's correct, this work is just for the mobile/sliding nav view. The desktop work shall be addressed here: https://warthogs.atlassian.net/browse/WD-12670